specification/nhs-website-content-api.yaml

```diff
@@ -18,8 +18,8 @@ info:
       * information about NHS services
 
     You cannot currently use this API to:
-      * access data from NHS Services
-      * access Ratings and Reviews of NHS organisations submitted to the NHS website
+      * [find services near you](https://www.nhs.uk/nhs-services/services-near-you/)
+      * [check ratings and reviews of organisations](https://developer.api.nhs.uk/nhs-api/documentation/comments)
```

Comment:
The 'Rating and Reviews' API is going to be moved to APIM soon too, not sure linking to the (old) Developer Portal is a good idea.

Reply:
Sorry, that should link to the nhs.uk page

Reply:
I might just remove it entirely